test/integration/Main.hs

```diff
@@ -1 +1,66 @@
-{-# OPTIONS_GHC -F -pgmF hspec-discover #-}
+module Main where
+
+import Control.Concurrent
+    ( threadDelay )
+import Control.Concurrent.MVar
+    ( newMVar )
+
+import Data.Aeson
+    ( Value )
+import Data.String
+    ( fromString )
+import Data.Text
+    ( Text )
+import Network.HTTP.Client
+    ( Manager, defaultManagerSettings, newManager )
+import Prelude
+import System.Process
+    ( proc, withCreateProcess )
+import Test.Hspec
+    ( beforeAll, describe, hspec )
+
+import qualified Data.Text as T
+
+import Test.Integration.Framework.DSL
+    ( Context (..)
+    , Scenarios
+    , expectError
+    , request
+    , request_
+    , scenario
+    , verify
+    )
+import Test.Integration.Framework.Request
+    ( RequestException (..) )
+
+import qualified Cardano.NetworkLayer.HttpBridgeSpec as HttpBridge
+
+withWallet :: ((Text, Manager) -> IO a) -> IO a
+withWallet action = do
+    let launch = proc "cardano-wallet-server" []
+        baseURL = T.pack "http://localhost:8090/"
+    manager <- newManager defaultManagerSettings
+    withCreateProcess launch $ \_ _ _ _ph -> do
+        threadDelay 1000000
+        action (baseURL, manager)
+
+main :: IO ()
+main = withWallet $ \http -> do
+    hspec $ do
+        describe "Cardano.NetworkLayer.HttpBridge" HttpBridge.spec
+
+        beforeAll (newMVar $ Context () http) $ do
+            describe "Integration test framework" dummySpec
+
+dummySpec :: Scenarios Context
+dummySpec = do
+    scenario "Try the API which isn't implemented yet" $ do
+        let endpoint = "api/wallets"
+        response <- request ("GET", fromString endpoint) Nothing
```

Comment:
Think we should use `OverloadedStrings` ultimately, as originally

Reply:
Fixed, thanks.